examples/controls_example/public/edit_example.tsx

```diff
@@ -177,10 +177,12 @@ export const EditExample = () => {
             </EuiButton>
           </EuiFlexItem>
         </EuiFlexGroup>
+        {/* Can't wrap the ControlGroupRenderer in the `EuiSkeletonTitle` component like it's
+             upposed to be used since it's the one that sets loading to false. */}
         {isLoading ? (
           <>
             <EuiSpacer />
-            <EuiLoadingContent lines={1} />
+            <EuiSkeletonTitle css={{ width: '100%' }} />
```

Comment:
I moved the `EuiForm` up one level so that the popover doesn't jump in size quite as much when loading:

| Before | After |
|--------|-------|
| ![May-02-2023 12-16-30](https://user-images.githubusercontent.com/8698078/235751601-eb7b20bd-501c-4f88-8362-f20444016674.gif) | ![May-02-2023 12-16-58](https://user-images.githubusercontent.com/8698078/235751586-1ee6000a-325e-4f28-a117-76f32684b03c.gif) |

Reply:
Definitely not a blocker / not a huge issue, but this looks like you may want `EuiSkeletonRectangle` here instead of title, if you're replacing an input element vs a title 🤔 You could also skip the custom `css` that way!

```tsx
<EuiSkeletonRectangle width="100%" height="2em" />
```

Note on the height: You could in theory grab the exact correct form row height in px from EUI's CSS-in-JS variables, but using `em`s instead is probably also an okay/safe guesstimation at the row height.

Reply:
Ooh, yeah - that's much cleaner, agreed :eyes:

![May-02-2023 15-54-18](https://user-images.githubusercontent.com/8698078/235794413-6f5a03da-0d35-4d13-a8f0-10504610b3d5.gif)

I think estimating `2em` is good enough, since this is just part of an example plugin :+1: